### ISSUE TITLE
Understanding the automl.show_models() output

### ISSUE THREAD

**Issue by RaviGprec** [automl/auto-sklearn#761]
**I'm confused with below outputs where trainin score is around 71% and testing score is around69%. But Dont see 71% or 69% is the show_models() output.** 
![image](https://user-images.githubusercontent.com/35065303/72041552-e339ef80-32d1-11ea-9b28-38e92d71f594.png)

**what is the first value in  automl.show_models() out put. What is this 0.26000 value ? it is accuracy of the model?**
[(0.260000, SimpleRegressionPipeline({'categorical_encoding:__choice__': 'one_hot_encoding', 'imputation:strategy': 'median', 'preprocessor:__choice__': 'polynomial', 'regressor:__choice__': 'liblinear_svr', 'rescaling:__choice__': 'none', 'categorical_encoding:one_hot_encoding:use_minimum_fraction': 'False', 'preprocessor:polynomial:degree': 2, 'preprocessor:polynomial:include_bias': 'True', 'preprocessor:polynomial:interaction_only': 'False', 'regressor:liblinear_svr:C': 103.11030493878376, 'regressor:liblinear_svr:dual': 'False', 'regressor:liblinear_svr:epsilon': 0.3011904329496452, 'regressor:liblinear_svr:fit_intercept': 'True', 'regressor:liblinear_svr:intercept_scaling': 1, 'regressor:liblinear_svr:loss': 'squared_epsilon_insensitive', 'regressor:liblinear_svr:tol': 1.0974773655165751e-05},
dataset_properties={
  'task': 4,
  'sparse': False,
  'multilabel': False,
  'multiclass': False,
  'target_type': 'regression',
  'signed': False})),

**Comment by Robotuks** [comment]
From what I understand, 0.26 is a weight for this method. So if you have multiple methods in your model, the output of model is `Output = weight1*method1_output +  weight2*method2_output + ...`

Is this a full output of your .show_models()? Because for me there are usually multiple methods and sum of their weights is equal to 1
Model itself does not save score because it depends on training data. To get score on current training data you can use `automl.sprint_statistics()` after fitting data.

**Comment by mfeurer** [comment]
Yep, that's totally corretc @Robotuks 

@RaviGprec you can also check a few earlier answers in #721 and #382 

I'm closing this for now as the question is answered. Please reopen if you have any further questions on this.
